.github/workflows/chromatic-main-and-prs.yml

```diff
@@ -0,0 +1,38 @@
+name: Chromatic Main and Active PRs only
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [assigned, ready_for_review, review_requested]
+
+jobs:
+  chromatic:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: install
+        run: yarn install --frozen-lockfile
+      - uses: chromaui/action-next@v1
+        with:
+          exitOnceUploaded: true
+          onlyChanged: true
+          traceChanged: true
+          diagnostics: true
+          # Only use this if you notice too many builds with TurboSnap exiting due to a global package.json, package-lock.json, yarn.lock
+          # Typically for use when you have an active repository with lots of rebasing or squash and merge commits
+          # Documentation https://www.chromatic.com/docs/turbosnap#avoid-re-testing-on-changes-to-package-control-files
+          # This pattern will ignore all of the following files (package.json src/app/package.json yarn.lock app/yarn.lock)
+          untraced: |
+            - '**/(package**.json|yarn.lock)'
+        env:
+          CHROMATIC_PROJECT_TOKEN: gcaw1ai2dgo
```

Comment:
```suggestion
```
Don't hardcode a project token, especially not in an example file because people are going to end up sending (test) builds there.

Reply:
@ghengeveld 
I changed the format to look for the GitHub secret but I don't have access to edit it.
Can you please add a project token value with the key CHROMATIC_PROJECT_TOKEN when you get a chance?

Reply:
Done